Added ALT_TAB macro example (#5593)
qmk/qmk_firmware: docs/feature_macros.md

This will clear all keys besides the mods currently pressed.

> ### Advanced Example: 

Reviewer: To make it more appropriate for the rest of the outlining
```suggestion
## Advanced Example: 
```

> #### Super ALT/TAB

Reviewer: ```suggestion
### Super ALT/TAB
```

Reviewer 2: ```suggestion
### Super Alt+Tab
```
`Alt+Tab` makes more sense imo; `ALT/TAB` makes it sound like it's either one or the other.

Reviewer: It's a little of both? :D

Reviewer 2: Alt↯Tab :smile:

    return MACRO_NONE;
};
```

#### Super ALT/TAB

> This marco will register `KC_LALT` and tap `KC_TAB` and wait for 1000ms. If the key is taped again it will send another `KC_TAP`, if there is no tap then unregister `KC_LALT`. Thus allowing you to cycle through windows. 

Reviewer 3: A couple typo fixes and some grammar polishing here.

```suggestion
This macro will register `KC_LALT` and tap `KC_TAB`, then wait for 1000ms. If the key is tapped again, it will send another `KC_TAB`; if there is no tap, `KC_LALT` will be unregistered, thus allowing you to cycle through windows. 
```

> ```

Reviewer: ```suggestion
```c
```
